Sources/Sentry/SentryScreenshotIntegration.m

```diff
@@ -59,8 +59,9 @@ - (void)uninstall
     [client removeAttachmentProcessor:self];
 }
 
-- (NSArray<SentryAttachment *> *)processAttachments:(NSArray<SentryAttachment *> *)attachments
-                                           forEvent:(nonnull SentryEvent *)event
+- (nonnull NSArray<SentryAttachment *> *)processAttachments:
```

Comment:
This class doesn't define `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END`. Adding these would require even more changes, and I want to keep this PR small. The same applies for `SentryViewHierarchyIntegration.m`.